brands/amenity/fast_food.json

```diff
@@ -1976,6 +1988,16 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Pizza Tutto": {
+    "countryCodes": ["es"],
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "Pizza Tutto",
```

Comment:
you'll want to find or create a wikidata entry for this or it won't show up in iD

Reply:
This explains why Multiópticas has Wikidata entries without Wikipedia links.

I created [Q72115627](https://www.wikidata.org/wiki/Q72115627), is it okay? I couldn't find much info about the company besides the location of its restaurants.

Reply:
Thanks, that looks ok 👍